Localization/de-DE/home.yaml

```
@@ -41,6 +53,49 @@ Für technische Hilfe: ",
 #URLs - Facebook and Twitter have localised name in certain languages:
 "foldingforum.org",
 "Facebook",
-"Twitter"
+"Twitter",
+
+#LATEST POSTS bold text
+#"LATEST POSTS",
+"NEUESTE ARTIKEL",
+
+# WHY WE NEED YOU bold text
+#"WHY WE NEED YOU",
+"WARUM WIR SIE BRAUCHEN",
+
+# Associated text
+#"Folding@home is a project focused on disease research. The problems we're solving require so many computer calculations and we need your help to find the cures!",
+"Folding@home ist ein Projekt, das sich auf die Erforschung von Krankheiten konzentriert. Die Problemlösungen erfordern eine so große Menge an Berechnungen, dass wir Ihre Hilfe brauchen, um Heilmittel zu finden!",
+
+#FIGHT DESIEASES TOGETHER WITH US bold text
+#"FIGHT DISEASES TOGETHER WITH US.",
+"BEKÄMPFT KRANKHEITEN GEMEINSAM MIT UNS",
```

```suggestion
"BEKÄMPFEN SIE KRANKHEITEN GEMEINSAM MIT UNS",
```

fits better into the formal style we decided to use

---

```suggestion
"BEKÄMPFEN SIE MIT UNS GEMEINSAM KRANKHEITEN",
```